tls/tls.md

```diff
@@ -116,8 +121,15 @@ message PublicKey {
 
 **TODO: PublicKey.Data looks underspecified. Define precisely how to marshal the key.**
 
-## Future Extensibility
+## SNI
+Clients MAY use the Server Name Indication (SNI) in the `ClientHello` as defined in [RFC 6066 §3](https://www.rfc-editor.org/rfc/rfc6066#section-3).
```

Comment:
How about mentioning the requirement of using the protocol id (i.e. `/tls/1.1.0`) here?